### PR TITLE
Compute challenge validation metrics every N rounds

### PR DESCRIPTION
**Pull request by psfoley**
This PR:
- Adds a new parameter (`challenge_metrics_validation_interval`) that determines how often the challenge metrics will be calculated. Higher settings for this value will result in shorter experiment runtime.
- Sets a default of computing challenge validation metrics every **two rounds**

### CODE REVIEW

File: Task_1/fets_challenge/experiment.py [hunk 1]
```diff
@@ -454,52 +456,71 @@ def run_challenge_experiment(aggregation_function,
...
+            experiment_results['hausdorff95_tc'].append(hausdorff95_tc)
+
+            logger.info(summary)
+        else:
+            # update the auc score
+            best_dice_over_time_auc += best_dice * round_time
+            summary = f'Skipped challenge validation metrics for round {round_num}'
+            logger.info(summary)
+
+        if save_checkpoints:
```
Comment (brandon-edwards) [inline Task_1/fets_challenge/experiment.py:512]: Looks like you execute the block to save checkpoints, two times (one over-writing the other)?

**Review by brandon-edwards — CHANGES_REQUESTED** [review]
I have left a comment for line 512 which seems to perform the same save to checkpoint as line 525.

File: Task_1/fets_challenge/experiment.py (hunk 1 above)
Comment (psfoley) [reply Task_1/fets_challenge/experiment.py:512]: Thanks @brandon-edwards  for catching this! This was a merge error. Fixing now

**Review by brandon-edwards — APPROVED** [review]
Looks good to me. Thanks Patrick

**Review by alexey-gruzdev — APPROVED** [review]

**Review by sarthakpati — APPROVED** [review]

**Pull request merged**